all/pom.xml

```diff
@@ -340,14 +445,6 @@
       <scope>compile</scope>
       <optional>true</optional>
```

Comment:
all is always included by the top level pom ... so how does this work with the different native artifacts for snapshots?